ui/tract-renderer/src/main/java/org/cru/godtools/tract/widget/PageContentLayout.java

```diff
@@ -406,6 +430,65 @@ protected LayoutParams generateLayoutParams(final ViewGroup.LayoutParams p) {
         return generateDefaultLayoutParams();
     }
 
+    //region Card Bounce Animation
+
+    /**
+     * This method will either create a new bounce Animation or start the one that is already created.
+     */
+    @UiThread
+    private void animateFirstCardView() {
+        if (mActiveCard != null) {
+            return;
+        }
+
+        for (int i = 0; i < getChildCount(); i++) {
+            final View child = getChildAt(i);
+            final LayoutParams lp = (LayoutParams) child.getLayoutParams();
+            switch (lp.childType) {
+                case CHILD_TYPE_CARD:
+                    bounceCardAnimation(child);
+                    return; // Stop after first Card View
+            }
+        }
+        stopBounceAnimation();
+
+    }
+
+    public void setAnimateCard(boolean animate) {
```

Comment:
rename to `setBounceFirstCard(boolean animate)`, so that we know this is related to the bounce animation.

Reply:
or something similar, just needs to be named something indicative of it relating to bouncing the first card.